Assets/Scripts/Players/Movement.cs

```diff
@@ -2,20 +2,51 @@
 
 public class Movement : MonoBehaviour {
     [SerializeField]
-    private float speed = 1;
-    private CharacterController controller; 
+    private Rigidbody2D rb;
+    // position
+    private float x, y = 0;
+    public bool isGrounded = false;
+    // force
+    private float gravity = 0.1f;
+    private float speed = 5f;
+    private float quadStrength = 15f;
 
-	// Use this for initialization
-	void Start () {
-        controller = GetComponent<CharacterController>();
+    void Start () {
+        rb = GetComponent<Rigidbody2D>();
     }
 
-    // Update is called once per frame
     void Update () {
-        Vector2 moveDirection = new Vector2(
-            Input.GetAxis("Horizontal"),
-            Input.GetAxis("Vertical")
-            ); 
-        controller.Move(moveDirection * speed * Time.deltaTime);
+
```

Comment:
Out of curiosity, why change from the simple `Input.GetAxis()` logic? Doesn't really matter in the long run. Was there it not working?